src/Modules/OrchardCore.Commerce.Payment.Stripe/Abstractions/IStripePaymentService.cs

```diff
@@ -72,5 +72,5 @@ string shoppingCartId
     /// <summary>
     /// Confirm the result of Stripe payment.
     /// </summary>
-    Task<PaymentOperationStatusViewModel> PaymentConfirmationAsync(string paymentIntent, string shoppingCartId);
+    Task<PaymentOperationStatusViewModel> PaymentConfirmationAsync(string paymentIntent, string shoppingCartId, bool needToJudgeIntentStorage = true);
```

Comment:
The name is not super clear, could you add a description as `/// <param>` above?

Reply:
will try to do it.